stdlib/ballerina-http/src/main/ballerina/net.http/http_circuit_breaker.bal

```diff
@@ -43,19 +57,43 @@ documentation {
     F{{failureThreshold}}  - The threshold for request failures. When this threshold is crossed, the circuit will trip.
                              The threshold should be a value between 0 and 1.
     F{{resetTimeout}} - The time period to wait before attempting to make another request to the upstream service.
-    F{{httpStatusCodes}} - Array of http response status codes which considered as failure responses.
+    F{{statusCodes}} - Array of http response status codes which considered as failure responses.
 }
 public struct CircuitBreakerConfig {
+    RollingWindow rollingWindow;
     float failureThreshold;
     int resetTimeout;
-    int [] httpStatusCodes;
+    int[] statusCodes;
+}
+
+documentation {
+    Represents Circuit Breaker rolling window configuration.
+
+    F{{timeWindow}}  - This is how long circuit breaker should keeps calculations to trip the circuit.
```

Comment:
Shall we change the wording here ?

"Time period in which the failure threshold is calculated"